RevenueCatUI/View+PresentPaywall.swift

```diff
@@ -16,6 +16,15 @@ import SwiftUI
 
 #if !os(macOS) && !os(tvOS)
 
+/// Presentation options to use with the View modifiers ``presentPaywallIfNeeded``.
```

Comment:
The `presentPaywallIfNeeded` reference won't be found like that. You'd have to do `View.presentPaywallIfNeeded` but even that probably won't work, so let's change it to
```suggestion
/// Presentation options to use with the View modifiers `.presentPaywallIfNeeded`.
```